packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -170,6 +164,11 @@ const ComboboxField = ({
     }
   }
 
+  if (customProps.placeholder) {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
did you forgot to remove this linter disable 🧐

Reply:
with `delete` also it is basically reassigning. Therefore linter disable is required.

Reply:
Why are you deleting `placeholder` from the custom props? The console would log a warning if it got applied to a dom element. Removing the prop would make this a major version bump, so it would be on consumers to not be passing in a prop that doesn't exist.

Reply:
I agree with Jaime, I don't think there is value in deleting the placeholder prop here. Consumers upgrading should be removing the prop from their code, deleting the prop will suppress any warnings that invalid props are being passed.